doc/index.rst

```diff
@@ -49,10 +49,12 @@ See :doc:`development` on setting up your environment and workflow.
 .. code-block:: python
 
    import fedmsg
-   fedmsg.publish(topic='testing', modname='test', msg={
+   fedmsg.publish(name='mybus', topic='testing', modname='test', msg={
```

Comment:
We could. It is a good idea actually.

So do you suggest I leave the original sample code and just note the `endpoint.py` entry based if the `name` parameter in `publish()` is used or not?